.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor
 dist
 getenvoy
+.idea
```

Comment:
nit: this should be ignored in user/developer's local gitignore file like `~/.gitignore`.